### PR TITLE
Docs update after migrating to TMS

### PR DESCRIPTION
**Pull request by gokulavasan**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -61,16 +59,17 @@ Deploy the JAR using the CDAP CLI::
...
 
 Sample Application Configuration file::
 
   {
     "config": {
-      "auditKafkaConfig": {
-        "zookeeperString": "hostname:2181/cdap/kafka"
+      "auditLogConfig": {
+        "limit" : 200,
+        "offsetData" :  "_auditOffset"
```
Comment (elfenheart) [inline README.rst]: Should this say `offsetData` or `offsetDataset`?

Comment (gokulavasan) [reply README.rst]: You are right. It should be ```offsetDataset```

**Comment by awholegunch** [comment]
@gokulavasan I have reviewed and edited the copy. Please check my changes carefully. LGTM.

**Review by awholegunch — COMMENTED** [review]
See commit.

**Comment by gokulavasan** [comment]
LGTM 👍

**Pull request merged**